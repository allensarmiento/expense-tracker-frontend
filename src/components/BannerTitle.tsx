import { ReactChild } from 'react';
import styled from 'styled-components';

const Title = styled.h3`
  margin-bottom: 1rem;
  text-align: center;
`;

type BannerTitleProps = {
  children?: ReactChild;
};

const BannerTitle = ({ children }: BannerTitleProps) => {
  return <Title>{children}</Title>;
};

export default BannerTitle;
