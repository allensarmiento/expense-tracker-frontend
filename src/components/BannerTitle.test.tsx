import { shallow } from 'enzyme';
import BannerTitle from './BannerTitle';

it('renders without crashing', () => {
  expect(shallow(<BannerTitle/>)).toMatchSnapshot();

  expect(shallow(<BannerTitle>Title</BannerTitle>)).toMatchSnapshot();
});
