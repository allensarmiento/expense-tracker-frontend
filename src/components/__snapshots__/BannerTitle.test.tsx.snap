// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `<styled.h3 />`;

exports[`renders without crashing 2`] = `
<styled.h3>
  Title
</styled.h3>
`;
